docs/Factory.md

```diff
@@ -0,0 +1,105 @@
+<h1 align="center">Fastify</h1>
+
+<a name="factory"></a>
+## Factory
+
+The Fastify module exports a factory function that is used to create new
+<a href="https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md"><code><b>Fastify server</b></code></a>
+instances. This factory function accepts an options object which is used to
+customize the resulting instance. This document describes the properties
+available in that options object.
+
+<a name="factory-http2"></a>
+### `http2` (Status: experimental)
+
+A truthy value that can be used to enable Node.js core's
+[HTTP/2](https://nodejs.org/dist/latest-v8.x/docs/api/http2.html) socket.
+
++ Default: `undefined` or `false`
+
+<a name="factory-https"></a>
+### `https`
+
+An object used to configure the server's listening socket for TLS. The options
+are the same as the Node.js core
+[`createServer` method](https://nodejs.org/dist/latest-v8.x/docs/api/https.html#https_https_createserver_options_requestlistener). When this property is a falsy value, the socket will not be
+configured for TLS.
+
+This option also applies when the
+<a href="https://github.com/fastify/fastify/blob/master/docs/Factory.md#factory-http2">
+<code><b>http2</b></code>
+</a> option is set.
+
++ Default: `undefined` or `false`
+
+<a name="factory-ignore-slash"></a>
+### `ignoreTrailingSlash`
+
+Fastify uses [find-my-way](https://github.com/delvedor/find-my-way) to handle
+routing. This option may be set to `true` to ignore traling slashes in routes.
+This option applies to *all* route registrations for the resulting server
+instance.
+
++ Default: `undefined` or `false`
+
+#### Example
+
+```js
+const fastify = require('fastify')({
+  ignoreTrailingSlash: true
+})
+
+// registers both "/foo" and "/foo/"
+fastify.get('/foo/', function (req, reply) {
+  res.send('foo')
+})
+
+// registers both "/bar" and "/bar/"
+fastify.get('/bar', function (req, reply) {
+  res.send('bar')
+})
+```
+
+<a name="factory-json-limit"></a>
+### `jsonBodyLimit`
+
+Defines the maximum payload, in bytes, the server is allowed to accept for
+`PUT` and `POST` bodies when using the internal `application/json` content type
```

Comment:
It applies to any request that isn't `GET` or `HEAD`, so you could just say "request bodies".